Makefile

```diff
@@ -17,3 +20,21 @@ check:
 # just fine: a bug in fuse-overlayfs?
 # test-in-container:
 # 	podman run --rm -ti -v $(CURDIR):/src:Z -w /src $(PACKIT_TESTS_IMAGE) make check
+
```

Comment:
Can the above, commented-out targets be removed now?

Reply:
yep, removed